src/Agent.Listener/_project.json

```diff
@@ -56,6 +56,7 @@
     "runtimes": {
         "win7-x64": { },
         "ubuntu.14.04-x64": { },
+        "ubuntu.16.04-x64": { },
```

Comment:
I think this was need to dev on ubuntu 16.  We also need to drop an ubuntu 16 tar gz which I do want to do ...